README.md

```diff
@@ -75,11 +87,16 @@ Default value is ./broker/queue/ for in and out and ./broker/processed/ for proc
 
 ## Environmental variables
 
-See [.env.example](.env.example) for environment variable documentation
+For detailed ducomentation about specific environment variables, see [.env.example](.env.example).
+
+For this project, `.env.example` must be copied both to the root directory as `.env` and to the `tilavarauspalvelu` directory (as `.env` as well). To reduce confusion and potential misconfiguration, it is advisable to only have `.env` as a file in the root directory. The `.env` file can then be symlinked to `tilavarauspalvelu` directory with the command:
```

Comment:
does it have to be in root dir? Or is just the `tilavarauspalvelu` dir enough?

Reply:
I did some building and yes, this has to be present both in root and `tilavarauspalvelu`.

Also, while building, I did discover the docker-compose version will not work with the current instructions. I'll update it to create a relative symlink instead of full path symlink!